src/controller_main.c

```diff
@@ -229,6 +229,8 @@ void reset_flow_level()
 
     //clear memory
     memzero(wallet.password_double_hash, sizeof(wallet.password_double_hash));
+    memzero(wallet_credential_data.passphrase, sizeof(wallet_credential_data.passphrase));
+
```

Comment:
no space. `cy_free` belongs to same action group i.e., cleaning memory allocations